code/modules/urist/modules/shipbattles/overmapships/components/advanced_components.dm

```diff
@@ -15,11 +15,11 @@
 	var/counterboarding = FALSE
 
 /datum/shipcomponents/teleporter/DoActivate()
-	if(mastership.boarding && !counterboarding)
+	if((mastership.boarding && !counterboarding) || mastership.target_ship.shipid != "nerva")
```

Comment:
Again, we could use `USING_MAP` here instead of using a constant. Or it is going to be a pain to debug why shields won't work on ships that is not Nerva, for example.

Reply:
Shouldn't make debugging harder as area checks later on currently use `USING_MAP` regardless. Was just a temporary constant to stop pirates fighting another ship from effecting the Nerva. Shouldn't be an issue as I don't believe we have any other maps that can do space combat, but just in case, patched in d7fc3d7